packages/core/src/base-axis-chart.ts

```diff
@@ -286,9 +286,14 @@ export class BaseAxisChart extends BaseChart {
 
 		// Align y axis title with y axis
 		yAxisTitleRef.attr("class", "y axis-label")
-		.attr("text-align", "center")
-		.attr("transform", `rotate(-90) translate(${yAxisTitleTranslate.x}, ${yAxisTitleTranslate.y})`)
-		.text(this.options.scales.y.title);
+		.attr("text-align", "center");
+
+		if (!this.options.rtl) {
+			yAxisTitleRef.attr("transform", `rotate(-90) translate(${yAxisTitleTranslate.x}, ${yAxisTitleTranslate.y})`);
+		} else {
+			yAxisTitleRef.attr("transform", `rotate(90) translate(${-yAxisTitleTranslate.x}, ${yAxisTitleTranslate.y})`);
+		}
+		yAxisTitleRef.text(this.options.scales.y.title);
```

Comment:
can this not be where we're doing:
```js
yAxisTitleRef.attr("class", "y axis-label")
			.attr("text-align", "center");
```

?

Reply:
What do you mean exactly?